tests/phpunit/tests/blocks/editor.php

```diff
@@ -454,6 +454,14 @@ public function test_wp_get_post_content_block_attributes() {
 		$this->assertSame( $attributes_with_layout, wp_get_post_content_block_attributes() );
 	}
 
+	public function test_wp_get_post_content_block_attributes_no_layout() {
+		$attributes_empty = array();
+
+		switch_theme( 'block-theme-post-content-default' );
+
+		$this->assertSame( $attributes_empty, wp_get_post_content_block_attributes() );
+	}
```

Comment:
Related to my above feedback, maybe add another test for `get_block_editor_settings()`, to check that it doesn't include the `postContentAttributes` field if the post block attributes aren't set.

Reply:
The [original test](https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/tests/blocks/editor.php#L443) already checks the return value for classic themes (when Post Content doesn't exist), do you think it would be worth splitting that check out into its own test?

Reply:
Oh got it! Yeah that's a good idea.

Reply:
Added a test in 0e75e53